integration/signer/tests/test-bypass-and-sign-with-kms.sh

```diff
@@ -22,14 +22,16 @@ set -eux
 GOOD_IMAGE_URL=gcr.io/$PROJECT_ID/signer-int-good-image:$BUILD_ID
 docker build --no-cache -t $GOOD_IMAGE_URL -f ./Dockerfile.good .
 
-trap 'delete_image $GOOD_IMAGE_URL'  EXIT
+clean_up() { ARG=$?; delete_image $GOOD_IMAGE_URL; exit $ARG;}
```

Comment:
What does the whole ARG thing do? Does it somehow preserve the exit code? Or am I totally misunderstanding it?

Reply:
Yeah, this tries to preserve the exit code.